assembl/static2/js/app/components/debate/common/editPostForm.jsx

```diff
@@ -165,9 +174,20 @@ class EditPostForm extends React.PureComponent<EditPostFormProps, EditPostFormSt
                 <Button className="button-cancel button-dark btn btn-default left" onClick={this.handleCancel}>
                   <Translate value="cancel" />
                 </Button>
-                <Button className="button-submit button-dark btn btn-default right" onClick={this.handleSubmit}>
+                <Button
+                  className="button-submit button-dark btn btn-default right"
+                  onClick={() => this.handleSubmit(PublicationStates.PUBLISHED)}
+                >
                   <Translate value="debate.post" />
                 </Button>
+                {this.props.draftable ? (
+                  <Button
+                    className="button-submit button-dark btn btn-default right btn-draft"
+                    onClick={() => this.handleSubmit(PublicationStates.DRAFT)}
+                  >
+                    <Translate value="debate.brightMirror.save" />
```

Comment:
maybe change msg id so that there is no ambiguity : `debate.brightMirror.saveDraft`